components/cam/src/physics/cam/micro_p3.F90

```diff
@@ -927,17 +908,16 @@ SUBROUTINE p3_main_main_loop(kts, kte, kbot, ktop, kdir, log_predictNc, dt, odt,
 
       ! Record microphysics tendencies for output:
       ! warm-phase process rates
-      p3_tend_out(k, 2) = qcacc   ! cloud droplet accretion by rain
-      p3_tend_out(k, 3) = qcaut   ! cloud droplet autoconversion to rain
-      p3_tend_out(k, 4) = ncacc   ! change in cloud droplet number from accretion by rain
-      p3_tend_out(k, 5) = ncautc  ! change in cloud droplet number from autoconversion
-      p3_tend_out(k, 6) = ncslf   ! change in cloud droplet number from self-collection  (Not in paper?)
-      p3_tend_out(k, 7) = nrslf   ! change in rain number from self-collection  (Not in paper?)
-      p3_tend_out(k, 8) = ncnuc   ! change in cloud droplet number from activation of CCN
-      p3_tend_out(k,10) = qcnuc   ! activation of cloud droplets from CCN
-      p3_tend_out(k,11) = qrevp   ! rain evaporation
-      p3_tend_out(k,13) = nrevp   ! change in rain number from evaporation
-      p3_tend_out(k,14) = ncautr  ! change in rain number from autoconversion of cloud water
+      p3_tend_out(k, 2) = qcacc     ! cloud droplet accretion by rain
+      p3_tend_out(k, 3) = qcaut     ! cloud droplet autoconversion to rain
+      p3_tend_out(k, 4) = ncacc     ! change in cloud droplet number from accretion by rain
+      p3_tend_out(k, 5) = ncautc    ! change in cloud droplet number from autoconversion
+      p3_tend_out(k, 6) = ncslf     ! change in cloud droplet number from self-collection  (Not in paper?)
+      p3_tend_out(k, 7) = nrslf     ! change in rain number from self-collection  (Not in paper?)
+      p3_tend_out(k, 8) = ncnuc(k)  ! change in cloud droplet number from activation of CCN
```

Comment:
Not sure we should pass ncnuc as an output from p3 because it is really an intent(in) variable now, right?

Reply:
@PeterCaldwell 
Yes it is 'intent(in)', otherwise we can save the 'npccn' at the interface.

Reply:
Ok, let's delete this line then. Note that there's something creepy happening wiht p3_tend_out where columns 8-10 aren't getting assigned any values(?). We should probably shift the numbering, but let's leave that for a future PR. I'll make a note of it.

Reply:
You got it.